game_cfg/game_cfg.h

```diff
@@ -0,0 +1,9 @@
+#ifndef GAME_CFG_H
+#define GAME_CFG_H
+
+#include "colors.h"
```

Comment:
The meta .h file which only includes the other .h files might be stupid. But does two things: reduce the number of include lines in the "real" header files, and provides a single place where I can document what's in the various cfg .h files. Again, driven by the choice to have multiples.

Reply:
I think this makes sense. It could probably live in a block at the top of game.h too. But I don't think there is anything wrong with a separate file.